src/storage/mod.rs

```diff
@@ -152,14 +151,21 @@ macro_rules! check_key_size {
 impl<E: Engine, L: LockManager> Storage<E, L> {
     /// Get concurrency of normal readpool.
     pub fn readpool_normal_concurrency(&self) -> usize {
```

Comment:
It is only used for load statistics. When the unified thread pool is enabled, the load statistics about the normal storage read pool will become zero. Currently let's just ignore this problem because this function will finally be removed together with the old read pools.